scrapers/il/events.py

```diff
@@ -37,8 +44,12 @@ def scrape_page(self, url):
             metainf[key] = value
 
         where = metainf["Location:"]
-        subject_matter = metainf["Subject Matter:"]
-        description = "{}, {}".format(ctty_name, subject_matter)
+
+        description = f"{chamber} {ctty_name}"
+        # Remove committee suffix from names
+        committee_suffix = " Committee"
+        if description.endswith(committee_suffix):
+            description = description[: -len(committee_suffix)]
```

Comment:
The solution is working well for all events except those with "Hearing Notice For" in the description. That string should be removed in a way that also does not result in the chamber being listed twice.

Two examples of event `description` using current solution:
- "Senate Hearing Notice For Environment and Conservation"
- "Senate Hearing Notice For Senate Special Committee on Pensions"

After update to this PR, these cases should instead be:
- "Senate Environment and Conservation"
- "Senate Special Committee on Pensions"

Reply:
Fixed with https://github.com/openstates/openstates-scrapers/pull/4582/commits/d4bfa46906d683c2357320c3bd1c23f0246fa7c9